scripts/utils/start-release.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2025 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+#Requires bash version to be >=4. Will add alternative for lower versions
+set -euo pipefail
+
+dry_run=false
+
+PARSED_OPTIONS=$(getopt -n "$0" -o d --long dry-run -- "$@")
+if [ $? -ne 0 ]; then
+    exit 1
+fi
+eval set -- "$PARSED_OPTIONS"
+
+while true; do
+    case "$1" in
+        -d|--dry-run)
+            dry_run=true
+            shift
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *)
+            break
+            ;;
+    esac
+done
+
+if ! current_version_v1=$(make "echo-v1"); then
```

Comment:
why is there so much logic split between this script and Python script? Neither of them is doing "one job".